src/main/java/com/imagination/cbs/controller/ReasonsForRecruitingController.java

```diff
@@ -0,0 +1,24 @@
+package com.imagination.cbs.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.imagination.cbs.dto.ReasonsForRecruitingDto;
+import com.imagination.cbs.service.impl.ReasonsForRecruitingServiceImpl;
+
+@RestController
+@RequestMapping("/reasonsForRecruiting")
```

Comment:
mapping uri should be spinal case as per standards, but if its camel case every where in application then i cant tell you to update every where, just sharing my thoughts ... have a look on https://blog.restcase.com/5-basic-rest-api-design-guidelines/